src/content/docs/apm/agents/php-agent/configuration/distributed-tracing-php-agent.mdx

```diff
@@ -0,0 +1,619 @@
+* [Infinite Tracing:](#infinite-tracing) An advanced alternative to standard distributed tracing
+* [Older APM agents:](#older-agents) Tracing options if you have older APM agents
+* [Manual instrumentation:](#manual-instrumentation) Tips if automatic instrumentation doesn't work
+
+<Callout variant="tip">
+  If you want to get more background before getting started, check out these topics:
+  * [How span sampling works](/docs/understand-dependencies/distributed-tracing/get-started/how-new-relic-distributed-tracing-works#sampling) explains distributed tracing options.
+  * [Impacts to APM](/docs/apm/distributed-tracing/getting-started/transition-guide-distributed-tracing) tells you what to expect if you are a current APM user but haven't set up distributed tracing.
+</Callout>
+
+## Guided installation for standard distributed tracing (recommended): [#quick-start-apm]
+
+This is the best approach to set up [standard distributed tracing](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#head-based) in either of these cases:
+
+* You haven't installed any APM agents for your services yet.
+* You want to instrument additional services with this agent.
+
+If you already have some services instrumented with this APM agent, and you want to include them in distributed tracing, you'll need to manually enable distributed tracing for each service. See [Options for older APM agents](#older-agents).
+
+You'll need a New Relic account to set up distributed tracing. If you don't already have one, you can quickly [create a free account](https://newrelic.com/signup).
+
+### Step 1. Identify services
+
+Figure out which services touch your request so you can instrument each of them to send trace data to New Relic.
+
+### Step 2. Instrument each service with an APM agent [#update-agents]
+
+For PHP services that are not currently instrumented with the New Relic agent, you can run our guided installation that both instruments your service and enables distributed tracing.
+
+You should run the guided installation for each service you want to instrument to ensure that each installation has a unique application name. If some of your services use other languages, simply run the [guided installation](/docs/distributed-tracing/enable-configure/quick-start) for those other languages.
+
+To start the guided installation, click the link below. When you are finished, repeat this process for other services that handle your requests.
+
+    <TechTileGrid>
+      <TechTile
+        name="APM: PHP"
+        to="https://one.newrelic.com/launcher/nr1-core.settings?pane=eyJuZXJkbGV0SWQiOiJ0dWNzb24ucGxnLWluc3RydW1lbnQtZXZlcnl0aGluZyJ9&cards[0]=eyJuZXJkbGV0SWQiOiJzZXR1cC1uZXJkbGV0cy5zZXR1cC1waHAtaW50ZWdyYXRpb24iLCJhY2NvdW50SWQiOjI2NDA0MDl9&platform[accountId]=2498654&platform[timeRange][duration]=1800000&platform[$isFallbackTimeRange]=true"
+        icon={<img src={phplogo} alt="PHP"/>}
```

Comment:
you may need to add the prop `variant="TechTile"` if the image looks weird, too big

Reply:
@roadlittledawn, I added this variant to the .NET image and it still looks very large. The C, PHP, and other tile images look OK. Is the syntax correct here?

```
<TechTile
  name="APM: .NET"
  variant="TechTile"
  to="https://one.newrelic.com/launcher/nr1-core.settings?pane=eyJuZXJkbGV0SWQiOiJ0dWNzb24ucGxnLWluc3RydW1lbnQtZXZlcnl0aGluZyJ9&cards[0]=eyJuZXJkbGV0SWQiOiJzZXR1cC1uZXJkbGV0cy5zZXR1cC1kb3RuZXQtaW50ZWdyYXRpb24iLCJhY2NvdW50SWQiOjI2NDA0MDl9&platform[accountId]=2498654&platform[timeRange][duration]=1800000&platform[$isFallbackTimeRange]=true"
  icon={<img src={dotnetlogo} alt=".NET"/>}
/>
```